docs/release_notes.rst

```diff
@@ -1,6 +1,25 @@
 Release notes
 =============
 
+2.5.0
+-----
+
+Bodhi 2.5.0 is a feature and bugfix release.
+
+
+Features
+^^^^^^^^
+* The ``bodhi`` CLI now supports the edit command for updates.
```

Comment:
It would be good to include a link to the issue here - you can see examples of that in the 2.4.0 release notes below.